Backbone.API/DevicesDbContextSeed.cs

```diff
@@ -32,15 +32,20 @@ private async Task SeedApplicationUsers(DevicesDbContext context)
         if (await context.Users.AnyAsync())
             return;
 
+        var basicTier = await context.Tiers.GetBasicTier(CancellationToken.None);
+        if (basicTier == null)
+            return;
```

Comment:
You should throw an exception if it doesn't exist, so that we know that something's wrong. You can stick with the one you threw in the previous version:
```c#
throw new Exception("Basic Tier was not found.");
```